opensrp-chw-core/src/test/java/org/smartregister/chw/core/dao/VisitDaoTest.java

```diff
@@ -190,6 +193,21 @@ public void testGetMedicalHistory() {
         Mockito.verify(database).rawQuery(Mockito.anyString(), Mockito.any());
     }
 
+    @Test
+    public void testGetVisitHistory() {
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`